### PR TITLE
DS-800 | @ericbakenhus | Results per page

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Adds events per page selector

# Review By (Date)
- Retro

# Criticality
- Low

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to /events/discovery
3. Verify there is an events per page selector and that it works as expected
4. Verify all the options (i.e. more than 10) can be see for each filter that contains a list of refinements (e.g. subject)

# Associated Issues and/or People
- DS-800

### CODE REVIEW

File: src/components/events-discovery/components/Chips/Chip.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const Chip = ({ attribute, label, remove = () => null }) => {
     <button
       type="button"
       onClick={remove}
-      className="su-flex su-items-center su-px-12 su-py-6 su-text-xl sm:su-text-2xl su-border-2 su-border-cardinal-red su-break-words su-text-cardinal-red hocus:su-text-white hocus:su-bg-cardinal-red-light su-rounded-full su-cursor-pointer"
+      className="su-flex su-items-center su-px-12 su-py-6 su-text-xl sm:su-text-2xl su-border-2 su-border-digital-red su-break-words su-text-digital-red hocus:su-text-white hocus:su-bg-cardinal-red-xdark su-rounded-full su-cursor-pointer su-transition-colors"
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/Chips/Chip.jsx:28]: Matching the colors used for other buttons and button links on the site.

File: src/components/events-discovery/components/EventsPerPage/EventsPerPage.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            root: '!su-text-16',
+            selected: '!su-bg-digital-red-light/10',
+          }}
+        >
+          {item.label}
+        </MenuItem>
+      ))}
+    </Select>
+  );
+};
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/EventsPerPage/EventsPerPage.jsx:58]: Custom hits per page component.

File: src/components/events-discovery/components/Facets/FacetCtx.jsx [hunk 3]
```diff
@@ -10,8 +10,8 @@ const knownFacets = [
   { attribute: 'format', label: 'Format', chip: 'Format', expanded: false },
   {
     attribute: 'subject',
-    label: 'Interest Areas',
-    chip: 'Interest',
+    label: 'Subject',
+    chip: 'Subject',
     expanded: false,
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/Facets/FacetCtx.jsx:15]: Label update.

File: src/components/events-discovery/components/Facets/FacetList.jsx [hunk 4]
```diff
@@ -46,6 +46,7 @@ export const FacetList = ({ attribute, label, subfilter = false }) => {
       {canRefine ? (
         <RefinementList
           attribute={attribute}
+          limit={1000}
```
Comment (ericbakenhus) [inline src/components/events-discovery/components/Facets/FacetList.jsx:49]: Show all the refinements.

**Review by sherakama — COMMENTED** [review]
Weeeee. Packing in the functionality now.

**Comment by ericbakenhus** [comment]
> Weeeee. Packing in the functionality now.

"QA" (Quick, Add more)

**Comment by sherakama** [comment]
> "QA" (Quick, Add more)

Lol, thats a new one to me.

**Pull request merged**